libfoundation/src/foundation-value.cpp

```diff
@@ -802,7 +802,7 @@ bool __MCValueInitialize(void)
 
 void __MCValueFinalize(void)
 {
-    for(uindex_t i = 0; i < sizeof(s_value_pools) / sizeof(s_value_pools[0]); i++)
+    for(index_t i = 0; i < sizeof(s_value_pools) / sizeof(s_value_pools[0]); i++)
```

Comment:
`comparison of unsigned expression < 0 is always false` (but I'm not sure why it thinks that `sizeof(s_value_pools) / sizeof(s_value_pools[0])` is always 0)